#!/usr/bin/env python
PACKAGE = "sr_piezo_feedback"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("contact_time", double_t, 0, "Fading time after contact",  0.25 , 0.25,   2)

gen.add("max_amplitude", double_t, 0, "Maximum amplitude on output signal",  1 , 0.5,   1)
gen.add("min_amplitude", double_t, 0, "Minimum amplitude on output signal",  0.2 , 0,   0.4)
gen.add("max_frequency", double_t, 0, "Maximum frequency on output signal",  80 , 30,   150)
gen.add("min_frequency", double_t, 0, "Minimum frequency on output signal",  1 , 0.1,   10)

gen.add("pst_saturation", double_t, 0, "PST_SATURATION",  550 , 500,   1200)


exit(gen.generate(PACKAGE, "sr_piezo_feedback_node", "SrPiezoFeedback"))